@xen-orchestra/disk-transform/src/Disk.mts

```diff
@@ -0,0 +1,93 @@
+export type DiskBlockData = Buffer
+export type DiskBlock = {
+  index: number // the index of the block. Offset in raw disk is index * blockSize
+  data: DiskBlockData // thue Buffer like data of this block. Must be blockSize length
```

Comment:
You can add JSDoc to your TS types.
```ts
... = {
  /*
  * The index of the block...
  */
  index: number
  ...
}
```

So you will have these comments in a more global way